project/src/components/card/card.jsx

```diff
@@ -1,14 +1,20 @@
-import React from 'react';
+import { React, useState } from 'react';
+import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import { AppRoute } from '../../const';
 import { getStarRating } from '../../common';
 import offerProps from '../offers/offer.props';
 
-function Card(props) {
-  const { offer } = props;
+function Card({offer, cardHoverHandler}) {
```

Comment:
```suggestion
function Card({ offer, cardHoverHandler }) {
```
 Где-то пробелы есть, где-то нету

Должен быть один стиль